apps/connect/src/globals.ts

```diff
@@ -1,15 +1,26 @@
+import { type IDocumentDriveServer } from 'document-drive';
+import {
+    type IDocumentAdminStorage,
+    type IDocumentOperationStorage,
+    type IDocumentStorage,
+    type IDriveOperationStorage,
+} from 'document-drive/storage/types';
 import type { ElectronAPI } from './preload.js';
 import type { IConnectCrypto } from './services/crypto/index.js';
 import type { IRenown } from './services/renown/types.js';
 import type { DocumentEditorDebugTools } from './utils/document-editor-debug-tools.js';
-export {};
 
 declare global {
     interface Window {
         electronAPI?: ElectronAPI;
         connectCrypto?: IConnectCrypto;
         renown?: IRenown;
         documentEditorDebugTools?: DocumentEditorDebugTools;
+        reactor?: IDocumentDriveServer;
```

Comment:
we are storing these references in the window. there are others, but those types are kept in the state package to avoid polluting this interface.